### ISSUE TITLE
Currently released version not passing R check

### ISSUE THREAD

**Issue by schuemie** [OHDSI/PatientLevelPrediction#397]
See GitHub Action results [here](https://github.com/OHDSI/PatientLevelPrediction/actions/runs/5298012045/jobs/9590132248)

Here are the error messages:
```
── Error ('test-sklearnClassifier.R:119:3'): RandomForest fit works ────────────
Error in `if (model$`__dict__`["oob_score_"] != reticulate::py_none()) {
    serialized_model["oob_score_"] <- model$oob_score_
    serialized_model["oob_decision_function_"] <- model$oob_decision_function_$tolist()
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction::fitPlp(...) at test-sklearnClassifier.R:119:2
 2.   ├─base::do.call(fun, args)
 3.   └─PatientLevelPrediction (local) `<fn>`(...)
 4.     ├─base::do.call(...)
 5.     └─PatientLevelPrediction (local) `<fn>`(...)
 6.       └─PatientLevelPrediction::sklearnToJson(...)
 7.         └─PatientLevelPrediction:::serializeRandomForest(model)
── Error ('test-sklearnClassifier.R:202:3'): Sklearn predict works ─────────────
Error in `if (model["meta"] == "decision-tree") {
    model <- deSerializeDecisionTree(model)
} else if (model["meta"] == "rf") {
    model <- deSerializeRandomForest(model)
} else if (model["meta"] == "adaboost") {
    model <- deSerializeAdaboost(model)
} else if (model["meta"] == "naive-bayes") {
    model <- deSerializeNaiveBayes(model)
} else if (model["meta"] == "mlp") {
    model <- deSerializeMlp(model)
} else if (model["meta"] == "svm") {
    model <- deSerializeSVM(model)
} else {
    stop("Unsupported model")
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction:::predictPythonSklearn(...) at test-sklearnClassifier.R:202:2
 2.   └─PatientLevelPrediction::sklearnFromJson(path = modelLocation)
── Error ('test-sklearnJson.R:27:3'): Decision tree to json is correct ─────────
Error in `if (model["meta"] == "decision-tree") {
    model <- deSerializeDecisionTree(model)
} else if (model["meta"] == "rf") {
    model <- deSerializeRandomForest(model)
} else if (model["meta"] == "adaboost") {
    model <- deSerializeAdaboost(model)
} else if (model["meta"] == "naive-bayes") {
    model <- deSerializeNaiveBayes(model)
} else if (model["meta"] == "mlp") {
    model <- deSerializeMlp(model)
} else if (model["meta"] == "svm") {
    model <- deSerializeSVM(model)
} else {
    stop("Unsupported model")
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction::sklearnFromJson(path) at test-sklearnJson.R:27:2
── Error ('test-sklearnJson.R:41:3'): Random forest to json is correct ─────────
Error in `if (model$`__dict__`["oob_score_"] != reticulate::py_none()) {
    serialized_model["oob_score_"] <- model$oob_score_
    serialized_model["oob_decision_function_"] <- model$oob_decision_function_$tolist()
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction::sklearnToJson(model, path) at test-sklearnJson.R:41:2
 2.   └─PatientLevelPrediction:::serializeRandomForest(model)
── Error ('test-sklearnJson.R:59:3'): Adaboost to json is correct ──────────────
Error in `if (model["meta"] == "decision-tree") {
    model <- deSerializeDecisionTree(model)
} else if (model["meta"] == "rf") {
    model <- deSerializeRandomForest(model)
} else if (model["meta"] == "adaboost") {
    model <- deSerializeAdaboost(model)
} else if (model["meta"] == "naive-bayes") {
    model <- deSerializeNaiveBayes(model)
} else if (model["meta"] == "mlp") {
    model <- deSerializeMlp(model)
} else if (model["meta"] == "svm") {
    model <- deSerializeSVM(model)
} else {
    stop("Unsupported model")
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction::sklearnFromJson(path) at test-sklearnJson.R:59:2
── Error ('test-sklearnJson.R:75:3'): Naive Bayes to json is correct ───────────
Error in `if (model["meta"] == "decision-tree") {
    model <- deSerializeDecisionTree(model)
} else if (model["meta"] == "rf") {
    model <- deSerializeRandomForest(model)
} else if (model["meta"] == "adaboost") {
    model <- deSerializeAdaboost(model)
} else if (model["meta"] == "naive-bayes") {
    model <- deSerializeNaiveBayes(model)
} else if (model["meta"] == "mlp") {
    model <- deSerializeMlp(model)
} else if (model["meta"] == "svm") {
    model <- deSerializeSVM(model)
} else {
    stop("Unsupported model")
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction::sklearnFromJson(path) at test-sklearnJson.R:75:2
── Error ('test-sklearnJson.R:92:3'): MLP to json is correct ───────────────────
Error in `if (model["meta"] == "decision-tree") {
    model <- deSerializeDecisionTree(model)
} else if (model["meta"] == "rf") {
    model <- deSerializeRandomForest(model)
} else if (model["meta"] == "adaboost") {
    model <- deSerializeAdaboost(model)
} else if (model["meta"] == "naive-bayes") {
    model <- deSerializeNaiveBayes(model)
} else if (model["meta"] == "mlp") {
    model <- deSerializeMlp(model)
} else if (model["meta"] == "svm") {
    model <- deSerializeSVM(model)
} else {
    stop("Unsupported model")
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction::sklearnFromJson(path) at test-sklearnJson.R:92:2
── Error ('test-sklearnJson.R:123:3'): SVM to json is correct ──────────────────
Error in `if (model["meta"] == "decision-tree") {
    model <- deSerializeDecisionTree(model)
} else if (model["meta"] == "rf") {
    model <- deSerializeRandomForest(model)
} else if (model["meta"] == "adaboost") {
    model <- deSerializeAdaboost(model)
} else if (model["meta"] == "naive-bayes") {
    model <- deSerializeNaiveBayes(model)
} else if (model["meta"] == "mlp") {
    model <- deSerializeMlp(model)
} else if (model["meta"] == "svm") {
    model <- deSerializeSVM(model)
} else {
    stop("Unsupported model")
}`: the condition has length > 1
Backtrace:
    ▆
 1. └─PatientLevelPrediction::sklearnFromJson(path) at test-sklearnJson.R:123:2
 ```

**Comment by egillax** [comment]
Managed to replicate locally. Culprit is a new `reticulate` version which changes the outcome object when doing comparisons with python objects. Will fix and do a patch release.

**Comment by egillax** [comment]
Fixed in version 6.3.3
